src/Uniphar.Dotnet.Cli/Uniphar.Dotnet.Cli.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net9.0</TargetFramework>
```

Comment:
Maybe multi-target for both `net8.0` and `net9.0` so we don't cause problems for devs that only have the .NET 8 SDK installed?